src/features/defi/components/DefiActionButtons.tsx

```diff
@@ -1,22 +1,22 @@
 import { Button, ButtonGroup } from '@chakra-ui/react'
 import {
-  EarnAction,
-  EarnParams,
-  EarnQueryParams
-} from 'features/earn/contexts/EarnManagerProvider/EarnManagerProvider'
+  DefiAction,
+  DefiParams,
+  DefiQueryParams
+} from 'features/defi/contexts/DefiManagerProvider/DefiManagerProvider'
 import { useTranslate } from 'react-polyglot'
 import { matchPath } from 'react-router-dom'
 import { useBrowserRouter } from 'context/BrowserRouterProvider/BrowserRouterProvider'
 
-export const EarnActionButtons = () => {
+export const DefiActionButtons = () => {
   const translate = useTranslate()
-  const { location, history } = useBrowserRouter<EarnQueryParams, EarnParams>()
-  const match = matchPath<EarnParams>(location.pathname, {
-    path: '/earn/:earnType/:provider/:action',
+  const { location, history } = useBrowserRouter<DefiQueryParams, DefiParams>()
+  const match = matchPath<DefiParams>(location.pathname, {
+    path: '/defi/:earnType/:provider/:action',
     exact: true
   })
 
-  const handleClick = (action: EarnAction) => {
+  const handleClick = (action: DefiAction) => {
     if (match?.params) {
       const { earnType, provider } = match.params
       history.replace({
```

Comment:
the pathname below on line 24 is wrong - it needs to be `/defi/${earnType}...` otherwise the deposit/withdraw buttons on the modal close the modal because it redirects to the wrong route